ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-commands.sh

```diff
@@ -34,7 +34,9 @@ tempest init openshift
 pushd ~/tempest/openshift
 
 TEMPEST_CONF_OVERRIDES=${TEMPEST_CONF_OVERRIDES:-}
-discover-tempest-config --os-cloud ${OS_CLOUD} --debug --create identity.v3_endpoint_type public ${TEMPEST_CONF_OVERRIDES}
+
+discover-tempest-config --os-cloud ${OS_CLOUD} --debug --create \
```

Comment:
Is the removal of `TEMPEST_CONF_OVERRIDES` intentional? It'll probably break the manila tests, which required extra config options, does it mess with the ironic tests?

Reply:
thanks!